pages/common/git-log.md

```diff
@@ -3,13 +3,9 @@
 > Show a history of commits.
 > More information: <https://git-scm.com/docs/git-log>.
 
-- Show the sequence of commits starting from the current one, in reverse chronological order:
+- Show the sequence of commits starting from the current one, in reverse chronological order (optionally specifying a path):
```

Comment:
Specifying a path to what?

I'm a bit confused here. Also, the optional bit with the square brackets makes this example quite complex - especially for the 1st example on the page. For the beginner I think I'd prefer to keep a basic `git log` without any arguments as the first example.

Reply:
Fixed I think -- Just removed the complex / confusing bits.